typescript/packages/subsurface-viewer/src/layers/grid3d/privateGrid3dLayer.ts

```diff
@@ -22,7 +22,7 @@ import fsShader from "./fragment.fs.glsl";
 import vsLineShader from "./vertex_lines.glsl";
 import fsLineShader from "./fragment_lines.glsl";
 import { localPhongLighting, utilities } from "../shader_modules";
-import { TGrid3DColoringMode } from "./grid3dLayer";
+import { IDiscretePropertyValueName, TGrid3DColoringMode } from "./grid3dLayer";
```

Comment:
There is a warning here: `Imports "IDiscretePropertyValueName" are only used as type`

Reply:
Fixed. IDiscretePropertyValueName is imported as type